### ISSUE TITLE
Custom select  inconsistent paddings 

### ISSUE THREAD

**Issue by GeoSot** [twbs/bootstrap#27399]
on [_variables.scss ](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L522)line 522 and 523

```
$custom-select-padding-y:  .375rem !default;
$custom-select-padding-x: .75rem !default;
```

May i suggest a small change? In order to follow input sizes changes, it could be like

```
$custom-select-padding-y:  $input-padding-y  !default;
$custom-select-padding-x:  $input-padding-x  !default;
```

**Comment by alecpl** [comment]
Good point, but it should be `$input-btn-padding-y` and `$input-btn-padding-x` for consistency with *-lg and *-sm variants.

**Comment by alecpl** [comment]
Or vice-versa, `$custom-select-padding-y-lg` should use `$input-padding-y-lg` and the same for -x-lg and -sm.

**Comment by mdo** [comment]
Was just resolved by #27415.
